docs/_templates/template_ft_in_action.md

```diff
@@ -0,0 +1,104 @@
+
+
+## Preparing data
+Outline where the data can be found, artifact names in hubble or if relevant, how a user might load their own custom data. 
+Add a link to supplementary dataset info, for example as a `See Also` {admonition}.
+If you are outlining how to preprocess a dataset from scratch, use {dropdown} to hide long code snippets.
+
+
+## Login to Finetuner
+Explain how the user can login to Finetuner and why this is necessary. Feel free to also link to other documentation as well.
+
+Example:
+
+"As explained in the [Login to Jina ecosystem](../2_step_by_step/2_3_login_to_jina_ecosystem.md) section, first we need to login to Finetuner:"
+```python
+import finetuner
+finetuner.login()
+```
+
+
+## Choosing the model
+Always show the available models in your guide. Then mention which model will be used in your fine-tuning task. 
+Feel free to add a `See Also` {admonition} for supplementary info on the model, perhaps a relevant paper or site.
+
+Example:
+
+"You can see all available models either in [the docs](../2_step_by_step/2_5_choose_back_bone.md) or by calling:"
+```python
+finetuner.describe_models()
+```
+
+```bash
+                                                                  Finetuner backbones                                                                   
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃                                            model ┃           task ┃ output_dim ┃ architecture ┃                                          description ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│                                         resnet50 │ image-to-image │       2048 │          CNN │                               Pretrained on ImageNet │
+│                                        resnet152 │ image-to-image │       2048 │          CNN │                               Pretrained on ImageNet │
+│                                  efficientnet_b0 │ image-to-image │       1280 │          CNN │                               Pretrained on ImageNet │
+│                                  efficientnet_b4 │ image-to-image │       1280 │          CNN │                               Pretrained on ImageNet │
+│                     openai/clip-vit-base-patch32 │  text-to-image │        768 │  transformer │ Pretrained on millions of text image pairs by OpenAI │
+│                                  bert-base-cased │   text-to-text │        768 │  transformer │       Pretrained on BookCorpus and English Wikipedia │
+│ sentence-transformers/msmarco-distilbert-base-v3 │   text-to-text │        768 │  transformer │           Pretrained on Bert, fine-tuned on MS Marco │
+└──────────────────────────────────────────────────┴────────────────┴────────────┴──────────────┴──────────────────────────────────────────────────────┘
```

Comment:
i would not do this in every guides, refer it to step-by-step should be enough

Reply:
+1